### PR TITLE
Add esgf-pyclient recipe

### PR DESCRIPTION
**Pull request by jarifibrahim**

### CODE REVIEW

**Comment by jarifibrahim** [comment]
@agoodman Please take a look at this too!

File: recipes/esgf-pyclient/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+test:
+  imports:
+    - pyesgf
+    - pyesgf.search
+    - pyesgf.security
+
+about:
+  home: http://esgf-pyclient.readthedocs.org
+  license: BSD 3-clause
```
Comment (jakirkham) [inline recipes/esgf-pyclient/meta.yaml]: So maybe make a comment like this for now in the recipe should be sufficient. Feel free to change the wording as you see fit.

```yaml
  # Please remember to add the license file after the PR is merged and included in a release.
  # https://github.com/ESGF/esgf-pyclient/pull/7
  # license_file: LICENSE
```

**Comment by jakirkham** [comment]
Thanks for fixing the indent to. As far as the license goes, let's just leave a comment like the one above and we should be good to merge. This can be addressed in a subsequent release.

**Comment by agoodm** [comment]
@jakirkham Done here and in #1614. Does this all look good to you now?

**Comment by jakirkham** [comment]
Yep, looks good. Thanks.

**Comment by jakirkham** [comment]
Soon you should get an email that invite you to join conda-forge, @agoodm @jarifibrahim . Once accepted you will be added to a team with the same name as this recipe. Those will give you permissions on the feedstock (repo) for this recipe.

Make sure when proposing any change that you go through the typical GitHub workflow of forking the feedstock and making changes in your fork that you PR back. Once merged CIs will build and deploy any changes you make.

Please let us know if you have any questions and welcome to conda-forge. :)

**Pull request merged**